### PR TITLE
Event test

### PR DESCRIPTION
**Pull request by maheshbhandari433**
MERN starter files.

### CODE REVIEW

File: client/public/logo192.png
Comment (Pavel-Kliukin) [inline client/public/logo192.png:1]: You should remove this logo

File: client/public/logo512.png
Comment (Pavel-Kliukin) [inline client/public/logo512.png:1]: This one also should be removed

File: client/public/manifest.json
Comment (Pavel-Kliukin) [inline client/public/manifest.json:1]: We don't need this file either

Comment (maheshbhandari433) [reply client/public/manifest.json:1]: logo images and manifest.json removed

**Review by Pavel-Kliukin — COMMENTED** [review]
Everything is good and approve 👍🏻

**Comment by Pavel-Kliukin** [comment]
Ok, it's good

**Pull request closed without merging**